core/commands/root.go

```diff
@@ -69,6 +69,9 @@ TOOL COMMANDS
     commands      List all available commands
 
 Use 'ipfs <command> --help' to learn more about each command.
+
+If you want to operate on a different repository than the default at ~/.ipfs,
+then prefix any ipfs command with IPFS_PATH=/path.
```

Comment:
I used your suggestion and updated the pull request.

> Sahib, do you know how to amend commits?

I just squashed my commits to one.
(Had to use ``git push --force`` but I guess nobody used that commit yet...)